examples/Component/LinearSystem/MatrixLinearSystem.scn

```diff
@@ -80,6 +80,10 @@
                 <MechanicalObject template="Vec3d" name="DOFs"/>
                 <SubsetMapping input="@../DOFs" output="@DOFs" indices="@box.indices"/>
             </Node>
+            <MatrixProjectionMethod areJacobiansConstant="true" mechanicalStates="@/rigidSections/green/a/DOFs @/rigidSections/green/a/DOFs"/>
```

Comment:
@damienmarchal I was not able to move this component in a header part because of links towards mechanical states that don't exist yet.